backend/migrations/20221021345323-fix-migrations.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db, callback) {
+    db.runSql("UPDATE migrations SET name = '/20221021345324-translation' WHERE name = '/22102134532452-translation'", [], callback);
```

Comment:
`db-migrate` seems to check the list of applied migrations before running the migrations per se, so if you apply this migration to the current database, the subsequent `translation` and `fix-content-source` migrations are listed in the table **twice**.

Not a huge issue, but they will be listed in the `migrations` table twice, and I'm not sure how this will work with future migrations.

Reply:
Should not really matter, and I'm not sure whether there's a better way to fix the issue.